plugin/openassetio_manager_bal/bal.py

```diff
@@ -343,9 +349,33 @@ def _copy_and_expand_trait_properties(entity_version_dict: dict, library: dict)
                     os.environ, **library["variables"]
                 )
 
+                subbed_val = trait_data[prop]
+
+                if subbed_val.startswith("file:/") or subbed_val.startswith("file:\\"):
```

Comment:
[question] Are we saying values like `file:///some/path is the path to the entity`, i.e. where it's not _really_ a path, is just undefined behaviour?

Reply:
I think we are, i wrote "url presenting" in the docs as a gesture to this. Not sure of an alternative that isn't too much effort/fiddly complexity for the value, given that using space as a delimeter is off the cards.